desk/app/chat.hoon

```diff
@@ -1702,6 +1702,7 @@
             mention=?
         ==
     ?.  ?|  =(net.dm %done)
+            =(net.dm %inviting)
```

Comment:
If we're transitioning into `%done` when receiving incoming activity, do we still need this here? Shouldn't the existing `=(net.dm %done)` be sufficient?

Reply:
you're right

```suggestion
```